main.c

```diff
@@ -48,12 +48,13 @@ static void usage(char* name)
 		"Usage: %s [option] action [data]\n"
 		"QR-Code decoder\n"
 		"\n"
-		"  -V, --version  display the version of this program\n"
-		"  -h, --help     print this help\n"
+		"  -V, --version       display the version of this program\n"
+		"  -h, --help          print this help\n"
 		"\n"
-		"  -e, --encode   encode to QR-code (data is a string)\n"
-		"  -f, --encfile  encode to QR-code (data is a file)\n"
-		"  -d, --decode   decode from QR-code (data is a file)\n"
+		"  -e, --encode        encode to QR-code (data is a string)\n"
+		"  -f, --encfile       encode to QR-code (data is a file)\n"
+		"  -d, --decode        decode from QR-code (data is a file)\n"
+		"  -n, --use-numbers   use the older but not so useful representation instead of ascii characters\n"
```

Comment:
I'm okay with changing the default output format. But let's use the real name of the original format:
```suggestion
		"  -n, --netpbm        output in the Netpbm P1 image format instead of providing a Unicode visualization\n"
```

Reply:
I set it to `use-netpbm`. I think it's a little more explicit.

Reply:
Good for me!